src/utils/isSupportedNetwork.ts

```diff
@@ -0,0 +1,9 @@
+import { ChainID } from "../enums"
+
+const supportedChainIds = [ChainID.Ethereum, ChainID.Ropsten]
```

Comment:
Based on comment above:

```suggestion
const supportedChainIds = [ChainID.Ethereum, ChainID.Ropsten, ChainID.Localhost]
```